packages/bodiless-core/src/components/PageEditor.tsx

```diff
@@ -49,6 +49,7 @@ const GlobalContextMenu: FC<Props> = observer(() => {
   const options = contextMenuOptions.filter(
     (op: TMenuOption) => op.global !== false,
   );
+  console.log('gcm', options);
```

Comment:
Are these two logs needed?

Reply:
@lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!